group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorService.java

```diff
@@ -268,7 +269,9 @@ public CompletableFuture<ConsumerGroupHeartbeatResponseData> consumerGroupHeartb
         ConsumerGroupHeartbeatRequestData request
```

Comment:
Should we also update the javadoc of all those methods to specify the contract a little more? I known that the old coordinator won't fully comply with this but as it will go away, it should not be a big deal.

Reply:
Not sure if I understand, should we specify in group coordinator interface or in group coordinator service? I made some change in the group coordinator interface.